src/http/representation/ResourceIdentifier.ts

```diff
@@ -14,3 +14,10 @@ export interface ResourceIdentifier {
 export function isResourceIdentifier(object: any): object is ResourceIdentifier {
   return object && (typeof object.path === 'string');
 }
+
+/**
+ * Factory function creating a resource identifier for convenience
+ */
+export function resourceIdentifier(resourcePath: string): ResourceIdentifier {
```

Comment:
```suggestion
export function createResourceIdentifier(resourcePath: string): ResourceIdentifier {
```
(for consistency)

Reply:
Renamed factory function `resourceIdentifier` to `createResourceIdentifier` (https://github.com/solid/community-server/pull/1075/commits/461a9361621353f3af3b65d09dddb2d633e96efa)